src/views/Dashboard.vue

```diff
@@ -0,0 +1,141 @@
+				<template v-slot:avatar>
+					<ConversationIcon
+						:item="item"
+						:hide-favorite="true"
+						:hide-call="true" />
+				</template>
+			</DashboardWidgetItem>
+		</template>
+		<template v-slot:empty-content>
+			<EmptyContent icon="icon-talk">
+				{{ t('spreed', 'Join a conversation or start a new one') }}
+				<template #desc>
+					<p>{{ t('spreed', 'Say hi to your friends and colleagues!') }}</p>
+					<button>{{ t('spreed', 'Start a conversation') }}</button>
+				</template>
+			</EmptyContent>
+		</template>
+	</DashboardWidget>
+</template>
+
+<script>
+import { DashboardWidget, DashboardWidgetItem } from '@nextcloud/vue-dashboard'
+import ConversationIcon from './../components/ConversationIcon'
+import EmptyContent from '@nextcloud/vue/dist/Components/EmptyContent'
+import axios from '@nextcloud/axios'
+import { generateOcsUrl, generateUrl } from '@nextcloud/router'
+
+const ROOM_POLLING_INTERVAL = 30
+
+const propertySort = (properties) => (a, b) => properties.map(obj => {
+	let dir = 1
+	if (obj[0] === '-') {
+		dir = -1
+		obj = obj.substring(1)
+	}
+	return a[obj] > b[obj] ? dir : a[obj] < b[obj] ? -(dir) : 0
+}).reduce((p, n) => p || n, 0)
+
+export default {
+	name: 'Dashboard',
+	components: { DashboardWidget, DashboardWidgetItem, ConversationIcon, EmptyContent },
+	data() {
+		return {
+			roomOptions: [],
+			loading: true,
+		}
+	},
+	computed: {
+		callLink() {
+			return (conversation) => {
+				return generateUrl('/call/' + conversation.token)
+			}
+		},
+		/**
+		 * This is a simplified version of the last chat message.
+		 * Parameters are parsed without markup (just replaced with the name),
+		 * e.g. no avatars on mentions.
+		 * @returns {string} A simple message to show below the conversation name
+		 */
+		simpleLastChatMessage() {
+			return (lastChatMessage) => {
+				if (!Object.keys(lastChatMessage).length) {
+					return ''
+				}
+
+				const params = lastChatMessage.messageParameters
+				let subtitle = lastChatMessage.message.trim()
+
+				// We don't really use rich objects in the subtitle, instead we fall back to the name of the item
+				Object.keys(params).forEach((parameterKey) => {
+					subtitle = subtitle.replace('{' + parameterKey + '}', params[parameterKey].name)
+				})
+
+				return subtitle
+			}
+		},
+		getWidgetItem() {
+			return (conversation) => {
+				return {
+					targetUrl: generateUrl(`/call/${conversation.token}`),
+					mainText: conversation.displayName,
+					subText: this.simpleLastChatMessage(conversation.lastMessage),
+					conversation,
+				}
+			}
+		},
+	},
+	beforeMount() {
+		this.fetchRooms()
+		// FIXME: reduce interval if user not active
+		setInterval(() => this.fetchRooms(), ROOM_POLLING_INTERVAL * 1000)
+	},
+	methods: {
+		fetchRooms() {
+			axios.get(generateOcsUrl('/apps/spreed/api/v1', 2) + 'room').then((response) => {
+				const rooms = response.data.ocs.data
+				rooms.sort(propertySort(['-hasCall', '-unreadMention', '-lastActivity']))
```

Comment:
So unread mentions are sorted top, but there is no visual indicator, which might be very confusing?